azure/virtualmachine/client.go

```diff
@@ -14,72 +14,81 @@ import (
-	azureRoleURL             = "services/hostedservices/%s/deployments/%s/roles/%s"
-	azureOperationsURL       = "services/hostedservices/%s/deployments/%s/roleinstances/%s/Operations"
-	azureCertificatListURL   = "services/hostedservices/%s/certificates"
-	azureRoleSizeListURL     = "rolesizes"
+	azureXmlns                        = "http://schemas.microsoft.com/windowsazure"
+	azureDeploymentListURL            = "services/hostedservices/%s/deployments"
+	azureHostedServiceListURL         = "services/hostedservices"
+	deleteAzureHostedServiceURL       = "services/hostedservices/%s?comp=media"
+	azureHostedServiceAvailabilityURL = "services/hostedservices/operations/isavailable/%s"
+	azureDeploymentURL                = "services/hostedservices/%s/deployments/%s"
+	deleteAzureDeploymentURL          = "services/hostedservices/%s/deployments/%s?comp=media"
+	azureRoleURL                      = "services/hostedservices/%s/deployments/%s/roles/%s"
+	azureOperationsURL                = "services/hostedservices/%s/deployments/%s/roleinstances/%s/Operations"
+	azureCertificatListURL            = "services/hostedservices/%s/certificates"
+	azureRoleSizeListURL              = "rolesizes"
 
 	osLinux                   = "Linux"
 	osWindows                 = "Windows"
 	dockerPublicConfigVersion = 2
 
-	provisioningConfDoesNotExistsError = "You should set azure VM provisioning config first"
-	invalidCertExtensionError          = "Certificate %s is invalid. Please specify %s certificate."
-	invalidOSError                     = "You must specify correct OS param. Valid values are 'Linux' and 'Windows'"
-	invalidDnsLengthError              = "The DNS name must be between 3 and 25 characters."
-	invalidPasswordLengthError         = "Password must be between 4 and 30 characters."
-	invalidPasswordError               = "Password must have at least one upper case, lower case and numeric character."
-	invalidRoleSizeError               = "Invalid role size: %s. Available role sizes: %s"
-	invalidRoleSizeInLocationError     = "Role size: %s not available in location: %s."
-	paramNotSpecifiedError             = "Parameter %s is not specified."
+	errParamNotSpecified            = "Parameter %s is not specified."
+	errProvisioningConfDoesNotExist = "You should set azure VM provisioning config first"
+	errInvalidCertExtension         = "Certificate %s is invalid. Please specify %s certificate."
+	errInvalidOS                    = "You must specify correct OS param. Valid values are 'Linux' and 'Windows'"
+	errInvalidPasswordLength        = "Password must be between 4 and 30 characters."
+	errInvalidPassword              = "Password must have at least one upper case, lower case and numeric character."
+	errInvalidRoleSize              = "Invalid role size: %s. Available role sizes: %s"
+	errInvalidRoleSizeInLocation    = "Role size: %s not available in location: %s."
+	errInvalidDnsLength             = "The DNS name must be between 3 and 25 characters."
```

Comment:
this will probably go unused with the removal of special logic but since it's const, you won't notice. just reminding :)